From 0407d0933e34459abde2ef21f6a710b29363eb73 Mon Sep 17 00:00:00 2001
From: Muhammad Husaini Zulkifli <muhammad.husaini.zulkifli@intel.com>
Date: Wed, 19 May 2021 23:04:50 +0800
Subject: [PATCH 108/122] igc: Set cycle time to max of 1s

Set the cycle time support to maximum value of 1s as per stated
in datasheet.

Signed-off-by: Muhammad Husaini Zulkifli <muhammad.husaini.zulkifli@intel.com>
---
 drivers/net/ethernet/intel/igc/igc_main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/intel/igc/igc_main.c b/drivers/net/ethernet/intel/igc/igc_main.c
index 26fa44512a4b..e38cbbf3340c 100644
--- a/drivers/net/ethernet/intel/igc/igc_main.c
+++ b/drivers/net/ethernet/intel/igc/igc_main.c
@@ -5540,13 +5540,13 @@ static int igc_tsn_clear_schedule(struct igc_adapter *adapter)
 	int i;
 
 	adapter->base_time = 0;
-	adapter->cycle_time = NSEC_PER_MSEC;
+	adapter->cycle_time = NSEC_PER_SEC;
 
 	for (i = 0; i < adapter->num_tx_queues; i++) {
 		struct igc_ring *ring = adapter->tx_ring[i];
 
 		ring->start_time = 0;
-		ring->end_time = NSEC_PER_MSEC;
+		ring->end_time = NSEC_PER_SEC;
 	}
 
 	return 0;
-- 
2.17.1

